agent-engines/gpu_worker/resource_monitor.py

```
@@ -0,0 +1,157 @@
+                    devices.append(
+                        {
+                            "device_id": index,
+                            "utilization_pct": float(utilization.gpu),
+                            "memory_used_mb": round(memory.used / (1024 * 1024), 2),
+                            "memory_total_mb": round(memory.total / (1024 * 1024), 2),
+                            "temperature_c": float(temperature),
+                        }
+                    )
+                return {"available": True, "devices": devices}
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)